### PR TITLE
updated mqtt topics and messages to match current payload

### PR DESCRIPTION
**Pull request by tobibechtold**

### CODE REVIEW

File: src/app/components/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -12,22 +12,41 @@ import { Score } from '../../models/score';
...
               private snackBar: MatSnackBar) {}
 
   ngOnInit() {
-    this.scoreService.score().subscribe(score => {
-      if (!this.isZeroZero(score)) {
-        this.showToast('GOOOOAL! Score is now' + ' ' + score.score[0] + ' - ' + score.score[1], 5000);
-      }
+    this.scoreService.score(0).subscribe(score => {
+      this._score.score[0] = score;
+      this.showScoreToast();
```
Comment (pfichtner) [inline src/app/components/dashboard/dashboard.component.ts]: - On new game the score will be '0'
- On reverting goals the team's score will decrease (goal reverted = no goal shot)

File: src/app/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -12,22 +12,41 @@ import { Score } from '../../models/score';
...
+    });
+
+    this.scoreService.score(1).subscribe(score => {
+      this._score.score[1] = score;
+      this.showScoreToast();
```
Comment (pfichtner) [inline src/app/components/dashboard/dashboard.component.ts]: - On new game the score will be '0'
- On reverting goals the team's score will decrease (goal reverted = no goal shot)

File: src/app/components/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -12,22 +12,41 @@ import { Score } from '../../models/score';
...
+        message = 'Its a draw';
+      } else {
+        message = 'Team ' + winner.winners[0] + ' won the game';
+      }
+      this.showToast(message, 15000);
     });
   }
 
+  private showScoreToast() {
+    if (!this.isZeroZero(this._score)) {
```
Comment (pfichtner) [inline src/app/components/dashboard/dashboard.component.ts]: - On new game the new team's scores are published separately (first team 0 then team  1, but order is not guranteed
- On reverting goals the team's score will decrease (goal reverted = no goal shot)

File: src/app/services/score.service.ts [hunk 4]
```diff
@@ -11,13 +10,13 @@ export class ScoreService {
 
   constructor(private mqttService: MqttService) { }
 
-  score(): Observable<Score> {
-    return this.mqttService.observe('game/score').pipe(switchMap(message =>  {
-      return of(JSON.parse(message.payload.toString()));
+  score(teamId: number): Observable<number> {
+    return this.mqttService.observe('game/score/' + teamId).pipe(switchMap(message =>  {
```
Comment (pfichtner) [inline src/app/services/score.service.ts]: please replace game/score by team/score (was wrong described in the message overview)

File: src/app/services/score.service.ts [hunk 5]
```diff
@@ -11,13 +10,13 @@ export class ScoreService {
...
+      return of(Number(message.payload.toString()));
     }));
   }
 
-  mockScoreFromMqtt(score: Score): Observable<void> {
-    return this.mqttService.publish('game/score', JSON.stringify(score));
+  mockScoreFromMqtt(teamId: number, score: number): Observable<void> {
+    return this.mqttService.publish('game/score/' + teamId, score.toString());
   }
```
Comment (pfichtner) [inline src/app/services/score.service.ts]: please replace game/score by team/score (was wrong described in the message overview)

File: .travis.yml [hunk 6]
```diff
@@ -5,7 +5,8 @@ dist: trusty
 sudo: required
 branches:
   only:
```
Comment (pfichtner) [inline .travis.yml]: Should not purging the whole "only" section fix the issue?

**Comment by pfichtner** [comment]
The toast message should be shown on receiving ```team/scored``` as this indicates that a team has scored (and this message is **not** send when a goal gets reverted).

**Comment by tobibechtold** [comment]
fixed wrong topic and made sure to reset the score as soon as gameover is sent. This doesn't impact the visible score on the UI but makes sure that the score on the toasts is always displayed correctly.

**Pull request merged**